### PR TITLE
Add Chips functionality and test

### PR DESCRIPTION
**Pull request by alorma**
Add chip tests:

*assertAnyChipSelected*: to check a `ChipGroup` has any chip selected.
*closeChip*: Chip action to act over close icon on chip

### CODE REVIEW

File: sample/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -9,6 +9,8 @@
     <item name="colorAccent">@color/colorAccent</item>
   </style>
 
+  <style name="MaterialAppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar"/>
```
Comment (alorma) [inline sample/src/main/res/values/styles.xml]: Chip needs MaterialComponent theme

File: library/src/main/java/com/schibsted/spain/barista/assertion/BaristaChipAssertions.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import androidx.annotation.IdRes
+import com.google.android.material.chip.ChipGroup
+import com.schibsted.spain.barista.assertion.BaristaAssertions.assertAny
+
+object BaristaChipAssertions {
+
+  @JvmStatic
+  fun assertAnyChipSelected(@IdRes viewId: Int) {
+    assertAny<ChipGroup>(viewId) {
+      it.checkedChipId != View.NO_ID
```
Comment (rocboronat) [inline library/src/main/java/com/schibsted/spain/barista/assertion/BaristaChipAssertions.kt]: Wow! Didn't know `NO_ID`. Thanks! 🙏

File: library/src/main/java/com/schibsted/spain/barista/interaction/BaristaChipInteractions.kt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.schibsted.spain.barista.interaction
+
+import com.schibsted.spain.barista.internal.performAction
+import com.schibsted.spain.barista.internal.util.resourceMatcher
+import com.schibsted.spain.barista.internal.viewaction.ChipViewActions.removeChip
+
+object BaristaChipInteractions {
+
+  @JvmStatic
+  fun closeChip(resId: Int) {
```
Comment (rocboronat) [inline library/src/main/java/com/schibsted/spain/barista/interaction/BaristaChipInteractions.kt]: I understand that you name it `close` instead of `remove` to avoid conflicting with the action itself...

File: sample/src/main/java/com/schibsted/spain/barista/sample/CameraActivity.kt [hunk 4]
```diff
@@ -29,6 +30,7 @@ class CameraActivity : AppCompatActivity() {
     }
   }
 
+  @SuppressLint("MissingSuperCall")
```
Comment (rocboronat) [inline sample/src/main/java/com/schibsted/spain/barista/sample/CameraActivity.kt]: This is not related to this PR... :·P

**Comment by rocboronat** [comment]
As the PR missed the breaking versions of the assertions tests, I added them :) Hope you don't mind this dirty hands writing in your PR! 😂 

By the way, I also changed a test method name and how to check that the label "the chip has been closed" appears. Very small changes that I think are easier to be done right now than asking you to do them 😄 

So, my feedback right now is merge at will! 🌐

**Comment by rocboronat** [comment]
I'm going to merge because I've asked Bernat in a private secret chat that only we know about
